lrpar/examples/calc_wasm/README.md

```diff
@@ -0,0 +1,13 @@
+# Parsing a simple calculator language in web assembly. 
+
+This contains a rudimentary port of the calculator example to web assembly.
+The intent is that this will work on various platforms *including* web assembly.
+
+The platforms this example is intended to run on include:
+
+* wasm32-unknown-unknown using wasm-bindgen
+* wasm32-wasip2 without wasm-bindgen
+* native targets.
+
```

Comment:
This should probably give instructions, and requirements.

Reply:
Should be fixed in [bfe3346](https://github.com/softdevteam/grmtools/pull/527/commits/bfe3346452c3ab83eef43c7bb46fa0a95a6e400a)